webserver/server/app/controllers/MintBurnHistoryController.ts

```diff
@@ -0,0 +1,151 @@
+import { Body, Controller, TsoaResponse, Res, Post, Route, SuccessResponse } from 'tsoa';
+import { StatusCodes } from 'http-status-codes';
+import tx from 'pg-tx';
+import pool from '../services/PgPoolSingleton';
+
+import type { ErrorShape } from '../../../shared/errors';
+import type { EndpointTypes } from '../../../shared/routes';
+import { Routes } from '../../../shared/routes';
+import { mintBurnRange, mintBurnRangeByPolicyIds } from '../services/MintBurnHistoryService';
+import type { MintBurnHistoryRequest, MintBurnHistoryResponse, MintBurnSingleResponse } from "../../../shared/models/MintBurn";
+import type { PolicyId } from "../../../shared/models/PolicyIdAssetMap";
+import type { ISqlMintBurnRangeResult, ISqlMintBurnRangeByPolicyIdsResult } from "../models/asset/mintBurnHistory.queries";
+
+const route = Routes.mintBurnHistory;
+
+@Route('asset/mint-burn-history')
+export class MintRangeController extends Controller {
+    @SuccessResponse(`${StatusCodes.OK}`)
+    @Post()
+    public async projectedNftRange(
```

Comment:
Missing documentation for the endpoints (see the other controllers for how to generate docs from comments)